forge-gui/res/cardsfolder/j/jaxis_the_troublemaker.txt

```diff
@@ -2,7 +2,9 @@ Name:Jaxis, the Troublemaker
 ManaCost:3 R
 Types:Legendary Creature Human Warrior
 PT:2/3
-A:AB$ CopyPermanent | Cost$ R T Discard<1/Card> | ValidTgts$ Creature.Other+YouCtrl | TgtPrompt$ Select another target creature you control | AddKeywords$ Haste | AddTriggers$ Dies | AddSVars$ TrigDraw | AtEOT$ Sacrifice | SorcerySpeed$ True | SpellDescription$ Create a token that's a copy of another target creature you control. It gains haste and "When this creature dies, draw a card." Sacrifice it at the beginning of the next end step. Activate only as a sorcery.
+A:AB$ CopyPermanent | Cost$ R T Discard<1/Card> | ValidTgts$ Creature.Other+YouCtrl | TgtPrompt$ Select another target creature you control | PumpKeywords$ Haste | AtEOT$ Sacrifice | SorcerySpeed$ True | RememberTokens$ True | SubAbility$ DBAnimate | SpellDescription$ Create a token that's a copy of another target creature you control. It gains haste and "When this creature dies, draw a card." Sacrifice it at the beginning of the next end step. Activate only as a sorcery.
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Triggers$ Dies | sVars$ TrigDraw | SubAbility$ DBCleanup
```

Comment:
is that sVars really needed? i thought we removed the need for svars

Reply:
Right, seems to work fine